samples/hssi/hssi_100g_cmd.h

```diff
@@ -56,6 +81,8 @@ class hssi_100g_cmd : public hssi_cmd
     , start_size_(64)
     , end_size_(9600)
     , end_select_("pkt_num")
+    , continuous_("off")
+    , monitor_("none")
```

Comment:
change name monitor_   to contmonitor_ 
 contmonitor_  change to int value
accept int value as user input

Reply:
sure we will update it.